radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -655,23 +652,6 @@ public Result mintTokens(String iso, BigDecimal amount) {
 		return execute(mintTokensAction);
 	}
 
-	/**
-	 * Mints an amount of new tokens and transfers it to another account
-	 *
-	 * @param iso The symbol of the token to mint
-	 * @param amount The amount in subunits to mint
-	 * @param toAddress The address that the minted tokens should be sent to
-	 * @return result of the transaction
-	 */
-	public Result mintAndTransferTokens(String iso, BigDecimal amount, RadixAddress toAddress) {
-		MintAndTransferTokensAction mintTokensAction = new MintAndTransferTokensAction(
-			RRI.of(getMyAddress(), iso),
-			amount,
-			toAddress
-		);
-		return execute(mintTokensAction);
-	}
```

Comment:
Am I right in assuming that, if the user wants to "mint and send", _(s)he_ is now responsible for creating the corresponding `AtomicAction` and executing it (preferably with this new `execute(Action)` method)?